astropy/io/fits/file.py

```diff
@@ -496,7 +497,7 @@ def _open_fileobj(self, fileobj, mode, overwrite):
             # to properly process the FITS header (and handle the possibility
             # of a compressed file).
             self._file.seek(0)
-        except (OSError,OSError):
+        except (OSError, OSError):
```

Comment:
🤔 that looks odd

Reply:
Oh, fun, I did not notice :D 
Comes from https://github.com/astropy/astropy/commit/be273588d96c2a3cd24a655abfafb368c4246519#diff-6eeb84ceec5958f743415dd8967222b9R470

Reply:
hmm, why am I not surprised. 😞

Reply:
No worries, it probably escaped your regex :)

Reply:
yeah, it wasn't  regrex, just a good old sed/awk one liner